straxen/plugins/x1t_cuts.py

```diff
@@ -198,7 +198,7 @@ class S1MaxPMT(strax.LoopPlugin):
     """Removes events where the largest hit in S1 is too large
     port from lax.sciencerun0.py"""
     depends_on = ('events', 'event_basics', 'peak_basics')
-    dtype = [('cut_s1_max_pmt', np.bool, 'S1 max PMT cut')] + strax.time_fields
+    dtype = [('cut_s1_max_pmt', np.bool_, 'S1 max PMT cut')] + strax.time_fields
```

Comment:
Please don't forget to bump the version

Reply:
Uh that is something I have not thought about. Are you afraid that decoding the data does not work any longer because we changed the dtype? I would have expected that np.bool and np.bool_ behaves the same. But I will do.